assets/quick-order-list.js

```diff
@@ -72,7 +72,7 @@ if (!customElements.get('quick-order-list')) {
       constructor() {
         super();
         this.cart = document.querySelector('cart-drawer');
-        this.quickOrderListId = `quick-order-list-${this.dataset.productId}`;
+        this.quickOrderListId = `${this.dataset.snippetId}-${this.dataset.productId}-${this.dataset.id}`;
```

Comment:
I don't think there is a need to add/have `this.dataset.snippetId`. We could have kept the hard coded string since the `dataset.id` is the differentiator. 

If you have two feat. collections on a same page that `snippedId` value is the same `quick-order-list-bulk`. 

So I think something we could remove and not need to have as a parameter when calling the snippet 🤔

Reply:
Good point 👍 trying to recall why i added it in the 1st place but it's not needed